File: ios/SelleoToast.m
//
//  SelleoToast.m
//  NativeTests
//
//  Created by Bartosz Boruta on 10/08/2021.
//

#import <Foundation/Foundation.h>
#import "React/RCTBridgeModule.h"

@interface RCT_EXTERN_MODULE(SelleoToast, NSObject)
  RCT_EXTERN_METHOD(showToast:(NSString *)message delay:(nonnull double *)delay)
@end

File: ios/SelleoToast.swift
//
//  SelleoToast.swift
//  NativeTests
//
//  Created by Bartosz Boruta on 10/08/2021.
//

import Foundation

@objc(SelleoToast)
class SelleoToast: NSObject {

  @objc
  func showToast(_ message: String, delay: Double = 1) {
    NSLog("Test1")
    print("Hello Swift1", delay)
    
    DispatchQueue.main.async{
      Toast.showToast(message, delay: delay)
    }
  }
}
